lib/quantizeAttributes.js

```diff
@@ -0,0 +1,348 @@
+'use strict';
```

Comment:
How hard is it to get test coverage for this file to 100%? https://coveralls.io/builds/6535609/source?filename=lib%2FquantizeAttributes.js